documentation/docs/05-misc/04-migrating-v3-to-v4.md

```diff
@@ -2,4 +2,56 @@
 title: Migrating from v3 to v4
 ---
 
-TODO
+This migration guide provides an overview of how to migrate from Svelte version 3 to 4. See the linked PRs for more detailed migration instructions. Use the migration script to migrate some of these automatically: `npx svelte-migrate svelte-4`
+
+If you're a library author, consider whether to only support Svelte 4 or if it's possible to support Svelte 3, too, after migrating - since most breaking changes don't affect many people, this may be easily possible. Also remember to update your `peerDependencies`.
```

Comment:
four commas in the first sentence is too many :laughing: maybe split it into two sentances
```suggestion
If you're a library author, consider whether to only support Svelte 4 or if it's possible to support Svelte 3 too. Since most of the breaking changes don't affect many people, this may be easily possible. Also remember to update the version range in your `peerDependencies`.
```